README.md

```diff
@@ -692,6 +692,7 @@ try {
     $log->info($e->getMessage());
     throw new InvalidLoginException();
 } catch (InvalidLoginException $e) {
+    sleep(5);
```

Comment:
Exponentially increases the amount of time a blind brute-force attack would take to work.